src/Sulu/Bundle/CustomUrlBundle/Resources/js/containers/Form/fields/CustomUrlsDomainSelect.js

```diff
@@ -26,19 +15,15 @@ class CustomUrlsDomainSelect extends React.Component<FieldTypeProps<string>> {
     };
 
     render() {
-        const {disabled, value} = this.props;
-
-        if (!this.webspace) {
-            return null;
-        }
+        const {disabled, formInspector, value} = this.props;
 
         return (
             <SingleSelect
                 disabled={!!disabled}
                 onChange={this.handleChange}
                 value={value}
             >
-                {this.webspace.customUrls.map(({url}) => (
+                {webspaceStore.getWebspace(formInspector.options.webspace).customUrls.map(({url}) => (
```

Comment:
would it not make sense to create a `@computed` value for webspaceStore.getWebspace(formInspector.options.webspace).customUrls ?

Reply:
Since we are not accessing it multiple times it should not make a big difference. Can still do it, would maybe also make that part easier to read.

Reply:
Is a computed recalculated in every render call? If not I would move them to @computed.

Reply:
No, it is not. It probably still doesn't make a huge difference, but I already moved it anyway :slightly_smiling_face: